src/Layer/C3DTilesLayer.js

```diff
@@ -15,6 +15,12 @@ export const C3DTILES_LAYER_EVENTS = {
      * @property type {string} on-tile-content-loaded
      */
     ON_TILE_CONTENT_LOADED: 'on-tile-content-loaded',
+    /**
+     * Fires when a tile is requested
+     * @event C3DTilesLayer#on-tile-requested
+     * @property type {string} on-tile-requested
+     */
```

Comment:
Could you specify what data is dispatched with the event ? (and could you also specify it for `ON_TILE_CONTENT_LOADED` please?)

Reply:
Actually in both case the data dispatched are not itowns class. For `ON_TILE_CONTENT_LOADED` this is the tile content something like that would be good ? @param {THREE.Object3D} tileContent - object3D of the tile, and for `ON_TILE_REQUESTED` this is a tile @param {object} metadata - tile requested ?

Reply:
yes perfect thanks :+1: